ui/components/app/alert-system/confirm-alert-modal/confirm-alert-modal.tsx

```diff
@@ -122,23 +122,37 @@ export function ConfirmAlertModal({
   ownerId,
 }: ConfirmAlertModalProps) {
   const t = useI18nContext();
-  const { alerts, unconfirmedDangerAlerts } = useAlerts(ownerId);
+  const { fieldAlerts, alerts, isAlertConfirmed } = useAlerts(ownerId);
 
   const [confirmCheckbox, setConfirmCheckbox] = useState<boolean>(false);
 
-  // if there are multiple alerts, show the multiple alert modal
+  const unconfirmedDangerAlerts = fieldAlerts.filter(
+    (alert) =>
+      !isAlertConfirmed(alert.key) && alert.severity === Severity.Danger,
+  );
```

Comment:
what if we moved this to useAlerts and rename to `unconfirmedDangerFieldAlerts`? This could be also be used in `ui/pages/confirmations/components/confirm/footer/footer.tsx`. A reason for this suggestion is to minimize confusion with the existing `unconfirmedDangerAlerts`  in useAlerts

Reply:
Great suggestion, done [here](https://github.com/MetaMask/metamask-extension/pull/27205/commits/15ec6140ec31e37c84c8ec1351b2060f54fd16a2).